protos/local/common.proto

```diff
@@ -38,10 +38,11 @@ message NamespacedWorkflowExecution {
 // Defines how an activity or workflow should be retried in the event of failure, timeout, etc.
 message RetryPolicy {
     // Interval of the first retry. If backoff_coefficient is 1.0 then it is used for all
-    // retries.
+    // retries. If this is not specified, there will be *no retrying*. Lang SDKs should require
+    // this to be set.
     google.protobuf.Duration initial_interval = 1;
     // Coefficient used to calculate the next retry interval. The next retry interval is previous
-    // interval multiplied by the coefficient. Must be 1 or larger.
+    // interval multiplied by the coefficient. Must be 1 or larger. Defaults to 2.
```

Comment:
Yeah I just realized that this is unclear. I will remove the defaults from here because they may actually be different depending on what the policy is used for.